### ISSUE TITLE
RAPIDS 22.12 conda environment using old networkx<=2.6.3 which has a bug with matplotlib-3. 6

### ISSUE THREAD

**Issue by mauriciocramos** [rapidsai/integration#594]
Dears,

The [current stable rapids-22.12 conda installation instruction](https://rapids.ai/start.html#get-rapids) is:
```
conda create -n rapids-22.12 -c rapidsai -c conda-forge -c nvidia  22.12 python=3.9 cudatoolkit=11.5
```
That instruction uses a recipe that has an upper bound on [`networkx>=2.5.1,<=2.6.3`](https://github.com/rapidsai/integration/blob/e4d8fbcd757f520a2234fdbc9312a42475a57b1f/conda/recipes/versions.yaml#L111).
```
mauricio@rig:~$ conda activate rapids-22.12
(rapids-22.12) mauricio@rig:~$ conda list "networkx|matplotlib"
# packages in environment at /home/mauricio/miniconda3/envs/rapids-22.12:
#
# Name                    Version                   Build  Channel
matplotlib-base           3.6.3            py39he190548_0    conda-forge
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
networkx                  2.6.3              pyhd8ed1ab_1    conda-forge
```
Notice rapids-22.12 has no upper bound to the current conda-forge matplotilib-3.6.

That specific networkx-2.6.3 has an old known [bug](https://github.com/networkx/networkx/issues/5938) related to the `matplotlib-3.6` which was fixed on [`networkx-2.8.6`](https://github.com/networkx/networkx/milestone/25?closed=1).

That bug prevents any user of rapids-22.12 to draw any networkx graphs.  I still don't even know why rapids uses networkx properly.  As far as I understood, its "just" related to [cugraph tests](https://github.com/rapidsai/cugraph/search?q=%22import+networkx%22)

If one wants to reproduce the networkx-2.6.3 error:
```
import matplotlib.pyplot as plt
import networkx as nx
G = nx.complete_graph(5)
nx.draw(G)
plt.show()
```

I already noticed that the `rapids-nightly` conda channel has a `rapids-23.02` recipe that already removed the networkx version upper bound, thus, solving this issue I raised.

As I can't use networkx with that stable recipe and also can't move to rapids-nightly, I kindly ask you to fix rapids-22.12 recipe by either removing the networkx version upper bound or at least raising that limit to networkx-2.8.6.

Thank you.

**Comment by raydouglass** [comment]
Hi @mauriciocramos! Unfortunately, we won't be able to make this change for `22.12`. However, this issue should be resolved in `23.02` as you noted. `23.02` is due for release on Feb 9th (See https://docs.rapids.ai/maintainers). So our recommendation is to update to `23.02` when it is released if you cannot use the nightlies now.

**Comment by mauriciocramos** [comment]
> `23.02` is due for release on Feb 9th (See https://docs.rapids.ai/maintainers).

Thank you for the link
